templates/generate_ood_mellon_metadata.sh.erb

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+MELLON_DIR="<%= scope['apache::params::httpd_dir'] -%>/mellon"
```

Comment:
```suggestion
MELLON_DIR="<%= scope['apache::httpd_dir'] -%>/mellon"
```
Ensure use correct value is someone overrides

Reply:
Also the defaults seem to have hardcoded paths while here it's based on Apache data.  I think it would be best to define something like `openondemand::mellon_dir` and make that used as base directory for all these files and paths.